app/routes/jsonPlaceHolder.js

```diff
@@ -0,0 +1,9 @@
+const { Router } = require('express');
```

Comment:
I think that this file willl be better call it `albums.js`. What do you think? 
If yo want you can check this [link](https://github.com/Wolox/tech-guides/blob/master/nodejs/docs/node-standard-and-best-practices.md#31--files)

Reply:
I didn't call `albums.js` because `JsonPlaceHolder` is the real API which have album endpoints. You don't aways going to use album endpoints, you could in a future require to use `/posts`, `/comments`, `/users`, `/photos` and those endpoints belong to `JsonPlaceHolder API`.

If you still believe that it's could be better call it to `albums.js`, no problem, I change it :)

Reply:
ok, At the moment we. only are using album endpoints, but I agree with you and in the future, we may use other resources.